e2e/tests/integration/api-splits.ts

```diff
@@ -60,8 +59,9 @@ describe('API splits', () => {
             .addManifest(apiManifest)
             .get('.notApiData',{ timeout: 5000 })
             .contains("10")
+            .get("[aria-expanded=\"false\"]")
```

Comment:
I'm not sure this will be the best thing to check for, as it just represents are tooltip that is unopened, and we'll soon have a lot of them. It would be better to test against the specific container that the charts will be inside I think.

Reply:
Basically we are checking this not exists at all .